deps/llvm.mk

```diff
@@ -433,9 +433,12 @@ $(eval $(call LLVM_PATCH,llvm-D57118-powerpc))
 $(eval $(call LLVM_PATCH,llvm-r355582-avxminmax)) # remove for 8.0
 endif # LLVM_VER
 
-# Independent to the llvm version add a JL prefix to the version map
-$(eval $(call LLVM_PATCH,llvm-symver-jlprefix)) # DO NOT REMOVE
-
+# Add a JL prefix to the version map. DO NOT REMOVE
+ifeq ($(LLVM_VER_SHORT), 6.0)
+$(eval $(call LLVM_PATCH,llvm6-symver-jlprefix))
```

Comment:
The way this part of the buildsystem works is by placing a .applied file for each patch. So contributers will see a failed patch (similar to the buildbots) since renaming the patch file will cause it to be applied twice.

Might be best not to rename it.

Reply:
Renamed patch file back to original for LLVM 6.